`grunt dist` fails

**Bug**

Running `grunt dist` throws an exception `error TS5011: File specification cannot contain multiple recursive directory wildcards ('**'): 'src/**/example/**/*.ts'.`

Due to changes in https://github.com/dojo/grunt-dojo2/pull/101

--- a/tasks/ts.ts
+++ b/tasks/ts.ts
@@ -5,7 +5,7 @@ export = function(grunt: IGrunt) {
 	const distDirectory = grunt.config.get<string>('distDirectory');
 	const defaultOptions: any = {
 		dist: {
-			exclude: ['tests/**/*.ts', 'src/**/example/**/*.ts'],
+			exclude: ['tests/**/*.ts', 'src/examples/**/*.ts'],
 			compilerOptions: {
 				outDir: distDirectory,
 				declaration: true,

